cfdm/read_write/netcdf/netcdfread.py

```diff
@@ -1683,6 +1693,76 @@ def _set_default_FillValue(self, construct, ncvar):
                 {"_FillValue": self.default_netCDF_fill_value(ncvar)},
             )
 
+    def _customize_auxiliary_coordinates(self, parent_ncvar, f):
+        """Create extra auxiliary coordinate constructs.
+
+        This method is primarily aimed at providing a customisation
+        entry point for subclasses. It is assumed that any new
+        constructs are set on the parent field or domain construct
+        inside this method.
+
+        .. versionadded:: TODOCFAVER
+
+        :Parameters:
+
+            parent_ncvar: `str`
+                The netCDF variable name of the parent variable.
+
+            f: `Field` or `Domain`
+                The parent field or domain construct.
+
+        :Returns:
+
+            `dict`
+                A mapping of netCDF variable names to newly-created
+                construct identifiers.
+
+        **Examples**
+
+        >>> n._customize_field_ancillaries('tas', f)
+        {}
+
+        >>> n._customize_field_ancillaries('pr', f)
+        {'tracking_id': 'fieldancillary1'}
+
+        """
+        return {}
+
+    def _customize_field_ancillaries(self, parent_ncvar, f):
+        """Create extra field ancillary constructs.
+
+        This method is primarily aimed at providing a customisation
```

Comment:
I'm with you! I wanted to use an `s`, but didn't want to (couldn't be bothered?) change other unrelated occurrences because it seemed unrelated to the PR. However, since you feel the same way I shall put in a commit for this (https://github.com/NCAS-CMS/cfdm/pull/255/commits/779419a53d62a1707cfc884ad7399463918b67fe),  and also one over at https://github.com/NCAS-CMS/cf-python/pull/630 which calls some if these. 

:)

Reply:
Glad I'm not the only weirdo to be uncomfortable with spelling inconsistencies! Thanks for updating those. Marking as resolved.